custom/enikshay/ucr/data_sources/episode_tasklist_v3.json

```diff
@@ -0,0 +1,2733 @@
+    "enikshay-test",
+    "enikshay-performance-test"
+  ],
+  "server_environment": [
+    "enikshay"
+  ],
+  "config": {
+    "referenced_doc_type": "CommCareCase",
+    "asynchronous": true,
+    "engine_id": "ucr",
+    "description": "",
+    "base_item_expression": {},
+    "table_id": "episode_task_list_v3",
+    "display_name": "episode (task list) v3",
+    "configured_filter": {
+      "type": "and",
+      "filters": [
+        {
+          "operator": "eq",
+          "expression": {
+            "datatype": null,
+            "type": "property_name",
+            "property_name": "type"
+          },
+          "type": "boolean_expression",
+          "comment": null,
+          "property_value": "episode"
+        },
+        {
+          "operator": "not_eq",
+          "expression": {
+            "datatype": "string",
+            "type": "named",
+            "name": "owner_id"
+          },
+          "type": "boolean_expression",
+          "comment": null,
+          "property_value": "_archive_"
+        },
+        {
+          "operator": "not_eq",
+          "expression": {
+            "datatype": "string",
+            "type": "named",
+            "name": "owner_id"
+          },
+          "type": "boolean_expression",
+          "comment": null,
+          "property_value": "_invalid_"
+        },
+        {
+          "filter": {
+            "operator": "in",
+            "expression": {
+              "datatype": "string",
+              "type": "named",
+              "name": "owner_id"
+            },
+            "type": "boolean_expression",
+            "comment": null,
+            "property_value": [
+              null,
+              ""
+            ]
+          },
+          "comment": null,
+          "type": "not"
+        }
+      ]
+    },
+    "configured_indicators": [
+      {
+        "comment": null,
+        "create_index": false,
+        "display_name": null,
+        "datatype": "string",
+        "expression": {
+          "datatype": "string",
+          "type": "named",
+          "name": "owner_id"
+        },
+        "transform": {},
+        "is_primary_key": false,
+        "is_nullable": true,
+        "type": "expression",
+        "column_id": "owner_id"
+      },
+      {
+        "comment": null,
+        "create_index": false,
+        "display_name": null,
+        "datatype": "string",
+        "expression": {
+          "value_expression": {
+            "datatype": null,
+            "type": "property_name",
+            "property_name": "name"
+          },
+          "type": "related_doc",
+          "related_doc_type": "Location",
+          "doc_id_expression": {
+            "type": "named",
+            "name": "owner_id"
+          }
+        },
+        "transform": {},
+        "is_primary_key": false,
+        "is_nullable": true,
+        "type": "expression",
+        "column_id": "owner_name"
+      },
+      {
+        "comment": "TODO: REMOVE?",
```

Comment:
should this be removed?

Reply:
(same question applies to all "todo" comments in here)

Reply:
Its present in the original data source as well `episode_task_listv2` so not changing it.